I/ImageStreamIO/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ImageStreamIO"
+version = v"1.0.3"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/milk-org/ImageStreamIO.git", "c4b3c531f4653f3b570345f43b2686cd25af0db8")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ImageStreamIO
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [Platform("x86_64", "linux")]
+
+# The products that we will ensure are always built
+products = Product[
+    LibraryProduct("libImageStreamIO", :libimagestreamio)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    BuildDependency(PackageSpec(name="CMake_jll", uuid="3f4e10e2-61f2-5801-8945-23b9d642d0e6"))
```

Comment:
Why CMake here?  We have CMake already in the build environment.  What you're installing here is probably not what you think it is.

Reply:
okay, good to know!